Gruntfile.js

```diff
@@ -496,7 +471,7 @@ module.exports = (grunt) => {
   grunt.registerTask('publishminor', ['default', 'test', 'release:minor']);
 
   // Create electron package
-  grunt.registerTask('package', ['clean:electron', 'clean:babel', 'babel:electron', 'copy:electron', 'electron']);
+  grunt.registerTask('package', ['clean:electron', 'clean:babel', 'babel:electron', 'electron']);
```

Comment:
We maybe conflating two PRs but we are doing babel for electron and removing babel in #1178.  

We want to be consistent either keep babel and keep wider support or just remove it.

Reply:
This PR is made to be merged before #1178 as it is simpler, in #1178 babel for electron is removed alongside babel for server.